roles/openstack-stack/templates/user_data.j2

```diff
@@ -0,0 +1,13 @@
+#cloud-config
+disable_root: true
+
+system_info:
+  default_user:
+    name: openshift
+    sudo: ["ALL=(ALL) NOPASSWD: ALL"]
+
```

Comment:
Potentially we could also put the docker-storage-setup file in user_data 

write_files:
- content: |
    DEVS='/dev/xvdb'
    VG=docker_vol
    DATA_SIZE=95%VG
    EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.basesize=3G"
  path: /etc/sysconfig/docker-storage-setup
  owner: root:root

Also, if we set the openshift_node_local_quota_per_fsgroup during install we could place the required fstab options here